app/mixins/mark-all-notifications.js

```diff
@@ -0,0 +1,14 @@
+import Mixin from '@ember/object/mixin';
```

Comment:
not a fan of mixins, they do allow for some code reuse, but they do make it way harder to reason about what happens here.

look at notifications.hbs -> there is some action called markAllAsRead
let's look it up in notifications.js oh it's not there
how does it work?
oh there is a mixin let's see what happens there...

oh, and is this mixin used only in one place? :confused: